templates/schedule.html

```diff
@@ -0,0 +1,137 @@
+    <td style="font-weight: bold">Workshop</td>
+  </tr>
+
+  <tr>
+    <td align="right">8:00</td>
+    <td colspan=2>Doors open &amp; Registration</td>
+  </tr>
+  <tr>
+    <td align="right">8:30</td>
+    <td colspan=2>Breakfast</td>
+  </tr>
+  <tr>
+    <td align="right">9:00</td>
+    <td colspan=2>Opening</td>
+  </tr>
+  <tr>
+    <td align="right">9:15</td>
+    <td colspan=2>Time for everybody to present their projects and/or find fellow Haskellers to work with</td>
+  </tr>
+  <tr>
+    <td align="right">9:30</td>
+    <td>Hack on</td>
+    <td>Beginner's Workshop</td>
+  </tr>
+  <tr>
+    <td align="right">11:00</td>
+    <td colspan=2>Keynote</td>
+  </tr>
+  <tr>
+    <td align="right">12:30</td>
+    <td colspan=2>Lunch</td>
+  </tr>
+  <tr>
+    <td align="right">14:00</td>
+    <td>Hack on</td>
+    <td>Beginner's Workshop (continued)</td>
+  </tr>
+  <tr>
+    <td align="right">15:30</td>
+    <td colspan=2>Coffee &amp; Cake</td>
+  </tr>
+  <tr>
+    <td align="right">17:00</td>
+    <td colspan=2>Official end</td>
+  </tr>
+
+  <!-- Day 2 -->
+  <tr>
+    <td colspan=3 style="border:1px solid black; padding-left:2pt; font-weight:bold; font-size: 125%; background-color: Plum">Saturday, Nov. 17th, 2018</td>
+  </tr>
+  <tr>
+    <td style="font-weight: bold">Time</td>
+    <td style="font-weight: bold">Hackathon</td>
+    <td style="font-weight: bold">Workshop</td>
+  </tr>
+
+  <tr>
+    <td align="right">8:30</td>
+    <td colspan=2>Doors open &amp; Breakfast</td>
+  </tr>
+  <tr>
+      <td/>
+      <td colspan=2>Hack on</td>
+  </tr>
+  <tr>
+    <td align="right">11:00</td>
+    <td colspan=2>Keynote</td>
+  </tr>
+  <tr>
+    <td align="right">12:30</td>
+    <td colspan=2>Lunch</td>
+  </tr>
+  <tr>
+    <td align="right">14:00</td>
+    <td>Hack on</td>
+    <td>Datatype-Generic Programming</td>
+  </tr>
+  <tr>
+    <td align="right">15:30</td>
+    <td colspan=2>Coffee &amp; Cake</td>
+  </tr>
+  <tr>
+    <td align="right">17:00</td>
+    <td colspan=2>Official end</td>
+  </tr>
+
+  <!-- Day 3 -->
+  <tr>
+    <td colspan=3 style="border:1px solid black; padding-left:2pt; font-weight:bold; font-size: 125%; background-color: Plum">Sunday, Nov. 18th, 2018</td>
+  </tr>
+  <tr>
+    <td style="font-weight: bold">Time</td>
+    <td style="font-weight: bold">Hackathon</td>
+    <td style="font-weight: bold">Workshop</td>
+  </tr>
+
+  <tr>
+    <td align="right">8:30</td>
+    <td colspan=2>Doors open</td>
+  </tr>
+  <tr>
+    <td/>
+    <td colspan=2>Hack on</td>
+  </tr>
+  <tr>
+    <td align="right">9:30</td>
+    <td colspan=2>Traditional Bavarian Breakfast</td>
+  </tr>
+  <tr>
+    <td align="right">11:00</td>
+    <td colspan=2>Keynote</td>
+  </tr>
+  <tr>
+    <td align="right">14:00</td>
+    <td colspan=2>Coffee &amp; Cake</td>
+  </tr>
+  <tr>
+    <td align="right">14:30</td>
+    <td colspan=2>Time for everybody to present their results (if you will)</td>
```

Comment:
Sounds very strange to me. Why not simply `Project presentations`?

Reply:
Yeah, why not. I'll change it.